demo/sizzle.sysl

```diff
@@ -0,0 +1,318 @@
+# This project tells our integration diagram what apps to include
+import mastercard.yaml as MasterCard
+
+# This app specifies the projects to include, it is optional and can be left out
+Sizzle[~project]:
+    @contact.name = "Jimmy Smith"
+    @description =:
+        | Sizzle is a Gourmet Sausage Restaurant.
+        |
+        | We aim to provide the truly Aussie sausage sizzle 
+        | to our customers.
+        |
+        | We will offer 90 minute table allocations and 
+        | comply with all directions and safety procedures 
+        | implemented by the state government.
+        |
+        | You can dine-in, take away, or shop online.
+    @version = "1.0.0"
+
+    Frontend:
+        Application
+    Backend:
+        BFF
+        Database
+
+
+Dine-in Customer [~human]:
```

Comment:
Love the modelling of the customers, this is a great example for documenting use-cases!

Reply:
@orlade-anz might be interested to see this also